spec/models/event_spec.rb

```diff
@@ -82,4 +82,31 @@
       end
     end
   end
+
+  describe '.duration' do
+    subject(:event_duration) { event.reload.duration }
```

Comment:
Didn't know that we could be using build this entire time. Nice one! I've changed this file over and will begin using this in all my new work in the future.